visualization-requirements.typ

```diff
@@ -39,3 +39,14 @@ A figma mockup and description of the default visualization:
 
 This figure represents the default case of one balancer being active, and no specific views/groupings (such as region) being active. As new monoliths are instantiated, they will appear on screen tethered to the balancer. As new rooms are instantiated, they will appear on-screen alongside the client that created them tethered to the appropriate monolith. Clients will not appear on-screen until they are connected to a room. When a monolith goes offline, the node representing it and any clients connected to that monolith should dissappear The room(s) connected to the (now disconnected) monolith should remain on-screen for some predetermined amount of time in the case a client connected to that room reconnects so it can be tethered to a new monolith.
 
+== Join Process
```

Comment:
nit: Could use a better name. This doesn't really show a step-by-step process, as the name would imply.